search-commons/src/main/java/no/unit/nva/search2/ResourceAwsClient.java

```diff
@@ -146,9 +147,10 @@ private OpenSearchSwsResponse handleResponse(HttpResponse<String> response) {
     }
 
     private MultiMatchQueryBuilder multiMatchQuery(ResourceAwsQuery query) {
-        var field = query.getValue(FIELDS).toString();
+        var fields = extractFields(query.getValue(FIELDS).toString());
+        var value = escapeSearchString(query.getValue(SEARCH_ALL).toString());
```

Comment:
query values should not be modified, only when values are transformed into query